orange/src/main/java/com/hartwig/hmftools/orange/algo/linx/LinxReportableClusters.java

```diff
@@ -30,6 +30,14 @@ public static Set<Integer> findReportableClusters(@NotNull LinxData linx)
         svIds.stream().filter(svToCluster::containsKey).map(svToCluster::get).forEach(clusterIds::add);
 
         linx.somaticDrivers().stream().filter(x -> x.clusterId() >= 0).forEach(x -> clusterIds.add(x.clusterId()));
-        return clusterIds;
+
+        return clusterIds.stream().filter(x -> !isSimpleCluster(linx, x)).collect(toSet());
+    }
+
+    private static boolean isSimpleCluster(@NotNull LinxData linx, int clusterId)
+    {
+        boolean hasSingleLink = !linx.clusterIdToChainCount().containsKey(clusterId) || linx.clusterIdToChainCount().get(clusterId) == 1;
+        boolean hasNoExons = !linx.clusterIdToExonCount().containsKey(clusterId) || linx.clusterIdToExonCount().get(clusterId) == 0;
```

Comment:
Based on the actual gene exon vis data, it seems that these maps are sparse (not all clusters are present in the maps). 

Is it still correct to assume that if a cluster does not exist in the mappings, it is a simple cluster?